scripts/xtensa-build-zephyr.py

```diff
@@ -323,6 +326,25 @@ def west_init_update():
 	# Do NOT "west update sof"!!
 	execute_command(["west", "update", "zephyr", "hal_xtensa"], timeout=300, cwd=west_top)
 
+def get_sof_version(abs_build_dir):
+	"""[summary] Get version string major.minor.micro of SOF firmware file.
+	When building multiple platforms from the same SOF commit, all platforms share the same version.
+	So for the 1st platform, generate the version string from sof_version.h and later plaforms will
+	reuse it.
+	"""
+	global sof_version
+	if sof_version:
+		return sof_version
+
+	versions = {}
+	with open(pathlib.Path(abs_build_dir, "zephyr/include/generated/sof_versions.h")) as hfile:
+		for hline in hfile:
+			words = hline.split()
+			if words[0] == '#define':
+				versions[words[1]] = words[2]
+	sof_version = versions['SOF_MAJOR'] + '.' + versions['SOF_MINOR'] + '.' + versions['SOF_MICRO']
```

Comment:
Newline 
```suggestion
	sof_version = versions['SOF_MAJOR'] + '.' + versions['SOF_MINOR'] +
	               '.' + versions['SOF_MICRO']
```
Again let `pylint` or `black` tell you the "right" way to format this.

Reply:
Thanks for the tips @marc-hb  Will fix all the identification issues.